Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -586,6 +586,31 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    // https://www.hackingwithswift.com/articles/141/8-useful-swift-extensions
+    /// SwifterSwift: Replaces a set number of occurrences of a string with replacement text.
+    ///
+    ///        "I swift swift".replacingOccurrences(of: "swift", with: "love", count: 1) -> "I love swift."
+    /// - Parameters:
+    ///     - search: the string to be replaced
+    ///     - replacement: the replacement string
+    ///     - maxReplacements: the maximum number of times to replace the string, which must be greater than 0
+    /// - Returns: The new string with the replacement text swapped the specified amount of times.
+    func replacingOccurrences(of search: String, with replacement: String, count maxReplacements: Int) -> String {
```

Comment:
This is just a suggestion, but I think you can get rid of `count`.
```suggestion
    func replacingOccurrences(of search: String, with replacement: String, maxReplacements: Int) -> String {
```

Reply:
For maximum flexibility I would recommend adding a `String.CompareOptions` parameter and passing that along to the underlying implementation.

That said, this seems like an _awfully_ contrived use case.